### DISCUSSION TITLE
Feature Request: Window Support Shelly 2pm

### DISCUSSION THREAD

**Discussion by Nastras** [mongoose-os-apps/shelly-homekit#1452] [Ideas]
Hi, would it be possible for you to support shelly 2pm as an additional mode window?

That would be great. Thanks a lot for your work!

**Comment by markirb** [comment]
I do not quite understand. Is roller shutter not the one you are looking for?

**Comment by Nastras** [comment]
I not only control window covers with the shelly but also an electric window. It would be great if you could add the Shelly as a window device. Then Siri would also know that it is a window and not a window cover.

I hope it is more understandable now?

Thank you 😉

**Comment by markirb** [comment]
I see. This is a new HAP type

**Comment by Nastras** [comment]
I don't think it's that new, with Velux Integra you can also control the windows in Homekit and the window cover as an extra device. This has been possible for quite a long time.

It would be very cool if you could implement this so that you can differentiate between windows and window covers.

**Comment by markirb** [comment]
It is not new. Just the support would be new.

**Comment by Nastras** [comment]
Ok, then I misunderstood you. Sorry.

Can you tell me if you implant it? I would then wait a little longer until I integrate the Shellys into Homekit.

**Comment by markirb** [comment]
I can not give you an ETA, I work on this firmware whenever I find time, and there are other issues here which want to be sorted out first

**Reply by Nastras** [reply]
No problem, I just wanted to know whether the idea is being pursued at all and has a chance of being implemented. 
No stress.

Many thanks for your work and effort👌
